### DISCUSSION TITLE
Web: How to use a different container other than "#main"?

### DISCUSSION THREAD

**Discussion by flosse** [DioxusLabs/dioxus#308] [Q&A] [answered]
I'm looking for something like `web::lanuch("my-container-id", app)`

**Comment by jkelleyrtp** [comment, marked as answer]
In the web config builder you can specify a different root name.

**Reply by flosse** [reply]
Thanks for that fast reply!
BTW: The [documentation](https://docs.rs/dioxus-web/latest/dioxus_web/struct.WebConfig.html#example) seems to be a but outdated,
instead of
```rust
dioxus::web::launch(App, |cfg| cfg.hydrate(true).root_name("myroot"))
```
it should be
```rust
dioxus::web::launch_cfg(app, |cfg| cfg.hydrate(true).rootname("myroot"))
```
